pdf_2_ics.py

```diff
@@ -216,14 +227,48 @@ def generate_ics(pdf_files, level, language):
     return True
 
 
-# Now we find the menu from the site
-url = "https://www.dpsnc.net/Page/7089"
-pdf_files = get_all_pdfs(url)
-# print(pdf_files)
+def parse_filename(filename):
+    if "Carb" in filename:
+        return False
+    if "Achievement" in filename:
+        return False
```

Comment:
My normal inclination would be to raise an exception in "exceptional" cases like this, rather than returning `False`, and catch it wherever this is called. Kinda weird to use exceptions for flow control like this, but it's standard practice in Python.

Reply:
Go ahead and fix with something more Pythonic if you have time. Good luck with the weather and remote school work @patrickkwang